isubit/luggage_roles
# Lugg 729 - Disable comments on page and people

Added strongarm configuration to disable comments by default.

Testing:
* Enable comments module
* Pull down this pull request
* Run drush fra -y
* Verify that this module turned off comments for pages.
